### DISCUSSION TITLE
Scale Worker Nodes of Global Service Cluster - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5279] [Announcements]
# Scale Worker Nodes of Global Service Cluster - DaoCloud Enterprise

DaoCloud Enterprise 5.0 is the next-generation PaaS platform, created with many leading cloud-native technologies, to easily promote and boost your digital business.

https://docs.daocloud.io/en/kpanda/best-practice/add-worker-node-on-global/

**Comment by peterchenhc** [comment]
# 《Add nodes to the global service cluster 》 
## The third step of this chapter needs attention:

**Custom Parameters**is added after the **check node** step.

**Reply by windsonsea** [reply]
Thanks, noted. This page is updated in PR #5283
